crates/noseyparker/src/github/models.rs

```diff
@@ -216,3 +216,23 @@ pub struct Repository {
     pub web_commit_signoff_required: Option<bool>,
     // pub security_and_analysis: Option<Option<Box<crate::models::MinimalRepositorySecurityAndAnalysis>>>,
 }
+
+// -------------------------------------------------------------------------------------------------
+// OrganizationShort
+// Defined as in: https://docs.github.com/en/enterprise-server@3.11/rest/orgs/orgs?apiVersion=2022-11-28#list-organizations
```

Comment:
This is also the same as this?

https://docs.github.com/en/rest/orgs/orgs?apiVersion=2022-11-28#list-organizations

Reply:
Replaced as yours is more general!